src/System.Management.Automation/FormatAndOutput/DefaultFormatters/FileSystem_format_ps1xml.cs

```diff
@@ -46,10 +46,10 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_FileSystemTypes(CustomC
                 TableControl.Create()
                     .GroupByProperty("PSParentPath", customControl: sharedControls[0])
                     .AddHeader(Alignment.Left, label: "UnixMode", width: 10)
-                    .AddHeader(Alignment.Left, label: "User", width: 16)
-                    .AddHeader(Alignment.Left, label: "Group", width: 16)
-                    .AddHeader(Alignment.Right, label: "LastWriteTime", width: 18)
-                    .AddHeader(Alignment.Right, label: "Size", width: 14)
+                    .AddHeader(Alignment.Right, label: "User", width: 10)
+                    .AddHeader(Alignment.Left, label: "Group", width: 10)
+                    .AddHeader(Alignment.Right, label: "LastWriteTime", width: 16)
+                    .AddHeader(Alignment.Right, label: "Size", width: 12)
```

Comment:
If we think about VMWare large files with size over 1Tb are a common occurrence. And they have PowerShell tools :smile:
```suggestion
                    .AddHeader(Alignment.Right, label: "Size", width: 14)
```

Reply:
And maybe use our new human formatting method we merged recently?

Reply:
I understand large VM disk could be over TB, like VHDX can be up to 64TB. But I don't think that's a common thing in most daily uses, also the list view could be helpful if the user really wants to see the length displayed on screen.

> And maybe use our new human formatting method we merged recently?

Size here should be accurate, so we should not use that.

Reply:
We can merge.